crates/hyvectl-commands/src/command.rs

```diff
@@ -33,6 +34,8 @@ pub enum Families {
     File(file::File),
     /// Prints the local Peer-id
     Whoami(whoami::Whoami),
+    /// Initialize a new hyveOS instance
```

Comment:
Maybe add a longer description here that clarifies that this command should only be used during installation (and that it probably needs sudo).